Demo/Demo/Features/FeatureBaseViewController.swift

```diff
@@ -0,0 +1,91 @@
+import UIKit
+
+class FeatureBaseViewController: UIViewController {
+
+    let containerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
```

Comment:
```suggestion
        view.backgroundColor = .systemBackground
```

We might as well use system colors when we can, since it will automatically support light & dark modes.